17thblog/logIn.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+include('config/config.php');
+include('config/init.php');
+
+echoHeader("Log In", "Log In");
+
+if(isset($_SESSION['Username']) && !empty($_SESSION['Username'])) {
+	echo "
+	<div class='textStyle form'>
+		**You are already logged in as $_SESSION[Username]!**
```

Comment:
From a UI standpoint, I'd probably want to be redirected to my account homepage if I'm logged in. This makes it look like an error occurred or something which isn't really true.